docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,16 +1,19 @@
-# Onboard public officers
+# Onboarding public officer using non-SE GSIB device
```

Comment:
pls remove this change as i have already updated based on Sowmya's review comments.